PackageExplorer/MefServices/PackageDownloader.cs

```diff
@@ -177,14 +177,20 @@ public async Task Download(string targetFilePath, SourceRepository sourceReposit
             {
                 try
                 {
+                    Console.WriteLine($"Downloading1: {packageIdentity.Id}");
```

Comment:
Should these be Console writes or use an ILogger, etc?

Reply:
They should not be there at all! Bad fixup, will fix. Thanks!